book_app.py

```diff
@@ -72,5 +80,6 @@ def menu():
     sys.exit()
 
 
+if __name__ == "__main__":
```

Comment:
Always use this block for any script that you write , never just call the `menu` function just like that